ui/plugins/auto_classification/panel.html

```diff
@@ -16,6 +16,7 @@
    ></th-autoclassify-toolbar>
 
 <div ng-switch="$ctrl.loadStatus">
+  <span ng-switch-when="job_pending">Job not complete, please wait</span>
```

Comment:
I sort of feel like a progress spinner like that sort of implies that there might be a state change soon, but for a pending job it might sit and spin for an hour or more.